plugin/lspconfig.lua

```diff
@@ -0,0 +1,85 @@
+local version_info = vim.version()
```

Comment:
Add a guard here, so it can be blocked from loading.
```suggestion
if vim.g.lspconfig ~= nil then
  return
end
vim.g.lspconfig = 1

local version_info = vim.version()
```

Reply:
Disabling those commands and defining your own instead is a valid thing to do :)